### PR TITLE
All instances of "Amplify app/application" have been changed to "Amplify Project"

### LINKED ISSUE #4690: Replace "Amplify app" with "Amplify project"
We realized that saying "Amplify app" was confusing—Amplify is not a platform like Android or iOS! "Amplify project" makes more sense.

Replace all instances of "Amplify app" in the docs with "Amplify project."

### PR DESCRIPTION
**Pull request by ledwinbatista**
…ify Project"

_Issue #, if available:_

fixes #4690 

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by katiegoines — APPROVED** [review]
LGTM

**Review by katiegoines — APPROVED** [review]

**Review by rachelnabors — CHANGES_REQUESTED** [review]
Let's consistently use all lowercase "project," as in "your Amplify project," not "your Amplify Project."

File: src/fragments/lib/auth/common/sms/flows.mdx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <Callout>
-Note: If you create or update an SMS MFA configuration for your Cognito user pool, the Cognito service will send a test SMS message to an internal number in order to verify your configuration. You will be charged for these test messages by Amazon SNS. 
+Note: If you create or update an SMS MFA configuration for your Cognito user pool, the Cognito service will send a test SMS message to an internal number in order to verify your configuration. You will be charged for these test messages by Amazon SNS.
 
 For information about Amazon SNS pricing, see [Worldwide SMS Pricing](https://aws.amazon.com/sns/sms-pricing/).
 </Callout>
 
 
-There are a few ways to integrate phone numbers into an Amplify app's sign-in and verification process. 
+There are a few ways to integrate phone numbers into an Amplify Project's sign-in and verification process.
```
Comment (katiegoines) [inline src/fragments/lib/auth/common/sms/flows.mdx:8]:
```suggestion
There are a few ways to integrate phone numbers into an Amplify project's sign-in and verification process.
```

File: src/pages/cli/hosting/hosting.mdx [hunk 2]
```diff
@@ -26,11 +26,11 @@ The AWS Amplify Console is a continuous deployment and hosting service for Ampli
 
 The AWS Amplify Console provides a Git-based workflow for building, deploying, and hosting your Amplify web app — both the frontend and backend — from source control. Once you connect a feature branch, all code commits are automatically deployed to an `amplifyapp.com` subdomain or your custom domain. [Get Started](https://docs.aws.amazon.com/amplify/latest/userguide/getting-started.html)
 
-Following are the concepts you would encounter when adding Amplify console as a hosting option for your Amplify app.
+Following are the concepts you would encounter when adding Amplify console as a hosting option for your Amplify project.
 
 ### Type of deployments
 
-If you select Amplify Console for hosting your Amplify App in the `amplify add hosting` flow, there are two stages you can select from as a part of the flow:
+If you select Amplify Console for hosting your Amplify Project in the `amplify add hosting` flow, there are two stages you can select from as a part of the flow:
```
Comment (katiegoines) [inline src/pages/cli/hosting/hosting.mdx:33]:
```suggestion
If you select Amplify Console for hosting your Amplify project in the `amplify add hosting` flow, there are two stages you can select from as a part of the flow:
```

File: src/pages/cli/hosting/hosting.mdx [hunk 3]
```diff
@@ -56,11 +56,11 @@ If you're hosting a Single Page Web App (SPA) with routing such as [`react-route
 
 ## Amazon S3 and Amazon Cloudfront
 
-The Amplify CLI provides you the option to manage the hosting of your static website using Amazon S3 and Amazon Cloudfront directly as well. Following are the concepts you would encounter when adding S3 & Cloudfront as a hosting option for your Amplify app.
+The Amplify CLI provides you the option to manage the hosting of your static website using Amazon S3 and Amazon Cloudfront directly as well. Following are the concepts you would encounter when adding S3 & Cloudfront as a hosting option for your Amplify project.
 
 ### Stages
 
-If you select Amazon S3 & Amazon Cloudfront for hosting your Amplify App in the `amplify add hosting` flow, there are two stages you can select from as a part of the flow:
+If you select Amazon S3 & Amazon Cloudfront for hosting your Amplify Project in the `amplify add hosting` flow, there are two stages you can select from as a part of the flow:
```
Comment (katiegoines) [inline src/pages/cli/hosting/hosting.mdx:63]:
```suggestion
If you select Amazon S3 & Amazon Cloudfront for hosting your Amplify project in the `amplify add hosting` flow, there are two stages you can select from as a part of the flow:
```

File: src/pages/cli/hosting/hosting.mdx [hunk 4]
```diff
@@ -75,7 +75,7 @@ For more  information of the Amazon S3 and Amazon CloudFront, check their docs:
 
 ### Cache Invalidation
 
-If you select Amazon S3 & Amazon Cloudfront for hosting your Amplify App in the `amplify add hosting` flow, the frontend build artifacts will be uploaded to the S3 hosting bucket, and then if Amazon CloudFront is enabled along with it, the `amplify publish` command executed with the `--invalidateCloudFront` or `-c` flag will send an invalidation request to the Amazon CloudFront service to invalidate its cache.
+If you select Amazon S3 & Amazon Cloudfront for hosting your Amplify Project in the `amplify add hosting` flow, the frontend build artifacts will be uploaded to the S3 hosting bucket, and then if Amazon CloudFront is enabled along with it, the `amplify publish` command executed with the `--invalidateCloudFront` or `-c` flag will send an invalidation request to the Amazon CloudFront service to invalidate its cache.
```
Comment (katiegoines) [inline src/pages/cli/hosting/hosting.mdx:78]:
```suggestion
If you select Amazon S3 & Amazon Cloudfront for hosting your Amplify project in the `amplify add hosting` flow, the frontend build artifacts will be uploaded to the S3 hosting bucket, and then if Amazon CloudFront is enabled along with it, the `amplify publish` command executed with the `--invalidateCloudFront` or `-c` flag will send an invalidation request to the Amazon CloudFront service to invalidate its cache.
```

**Review by katiegoines — CHANGES_REQUESTED** [review]
Made suggestions to address the changes @rachelnabors requested. Thanks!

**Review by katiegoines — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]
hacktober-amplify-2022 (1)

Thanks so much for contributing to our docs for Hacktoberfest! We appreciate all the care and effort you put into this PR!

**Pull request merged**